config.toml

```diff
@@ -276,6 +276,26 @@ memo_prefix = ''
 #   ['transfer', 'channel-0'],
 # ]
 
+# This section specifies the filters for incentivized packet relaying.
+# Default: no filters, will relay all packets even if they
+# are not incentivized.
+#
+# It is possible to specify the channel or use wildcards for the
+# channels.
+# The only fee which can be parametrized is the `recv_fee`.
+#
+# Example configuration of a filter which will only relay incentivized
+# packets, with no regards for channel and amount.
+#
+# [chains.packet_filter.min_fees.'*']
+# recv    = [ { amount = 0 } ]
+#
+# Example configuration of a filter which will only relay packets with
+# a specified minimum amount of fee if they are destined for 'channel-0'.
```

Comment:
Let's expand on this a little bit to explain that _either_ 20stake or 10uatom are needed for the packet to be relayed.